nightingale_body/src/nightingale_body/body_control.py

```diff
@@ -46,17 +50,55 @@ def _make_goal(self, jnt_name, jnt_pos, ee_vel=0.5, ee_accel=0.5, timeout=5):
         return goal
 
     def cmd_head(self, target_jnt_name, target_jnt_pos):
```

Comment:
cmd_head (as well as cmd_torso) should take only the target joint values. This is for consistency with arm joint control.

Reply:
I did this to separate the responsibility of getting and storing the desired values from calling the action client. The BodyControl class gets the joint names along with the desired values, so it seems to make sense to pass them from the BodyControl class rather than storing the joint names in the joint control class